client/my-sites/people/invite-people/index.jsx

```diff
@@ -342,7 +342,7 @@ class InvitePeople extends React.Component {
 							/>
 							<FormSettingExplanation>
 								{ translate(
-									'Enter up to 10 WordPress.com usernames or email addresses at a time.'
+									'If the user has an existing WordPress.com account, add their username. Otherwise, add their Email ID to send them an invite. You can add up to 10 people at a time.'
```

Comment:
Just a minor thing I noticed, but isn't "Email ID" a bit bizarre compared to the language usually used in Calypso of "email address"? It might be counter-intuitive when seeking clarity. 

```suggestion
									'If the user has an existing WordPress.com account, add their username. Otherwise, add their email address to send them an invite. You can add up to 10 people at a time.'
```

Reply:
@Aurorum Thanks for your review. `Email address` is indeed more speaking as `Email ID`. I've updated the PR accordingly.